homeassistant/components/zwave_js/climate.py

```diff
@@ -0,0 +1,335 @@
+"""Representation of Z-Wave locks."""
```

Comment:
Guessing you mean Climate instead locks :)

Reply:
indeed, thanks!